Mathlib/Logic/Basic.lean

```diff
@@ -1032,17 +1036,22 @@ theorem BEx.intro (a : α) (h₁ : p a) (h₂ : P a h₁) : ∃ (x : _) (h : p x
   ⟨a, h₁, h₂⟩
 #align bex.intro BEx.intro
 
-theorem ball_congr (H : ∀ x h, P x h ↔ Q x h) : (∀ x h, P x h) ↔ ∀ x h, Q x h :=
+theorem forall_mem_congr (H : ∀ x h, P x h ↔ Q x h) : (∀ x h, P x h) ↔ ∀ x h, Q x h :=
   forall_congr' fun x ↦ forall_congr' (H x)
-#align ball_congr ball_congr
+#align ball_congr forall_mem_congr
 
-theorem bex_congr (H : ∀ x h, P x h ↔ Q x h) : (∃ x h, P x h) ↔ ∃ x h, Q x h :=
+theorem exists_mem_congr (H : ∀ x h, P x h ↔ Q x h) : (∃ x h, P x h) ↔ ∃ x h, Q x h :=
   exists_congr fun x ↦ exists_congr (H x)
-#align bex_congr bex_congr
+#align bex_congr exists_mem_congr
 
-theorem bex_eq_left {a : α} : (∃ (x : _) (_ : x = a), p x) ↔ p a := by
+theorem exists_mem_eq_left {a : α} : (∃ (x : _) (_ : x = a), p x) ↔ p a := by
```

Comment:
```suggestion
theorem exists_eq_left {a : α} : (∃ (x : _) (_ : x = a), p x) ↔ p a := by
```

Reply:
Same here, this already exists at: https://leanprover-community.github.io/mathlib4_docs/Init/PropLemmas.html#exists_eq_left

Reply:
This one I think you can just deprecate since it's quite convoluted and simplifiable by simp already